src/ProductConstructionService/ProductConstructionService.Cli/Operations/DeploymentOperation.cs

```diff
@@ -75,9 +76,6 @@ public async Task<int> RunAsync()
 
             // Cleanup all revisions except the currently active one
             await CleanupRevisionsAsync(trafficWeights.Where(weight => weight != activeRevisionTrafficWeight));
-
-            // Finish current work items and stop processing new ones
-            await StopProcessingNewJobs();
```

Comment:
don't think this is correct. We should still tell PCS to stop processing new jobs, but just not wait for it to actually stop.

Reply:
The current revision of PCS will then finish it's currently working workitems, and stop.
When we're deactivating this revision later, we need to make sure that it actually stopped, so we'd wait there instead of here

Reply:
Good point, will update